tests/installation/partitioning_lvm_thin_provisioning.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: LVM thin provisioning setup
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use strict;
+use warnings;
+use base 'y2logsstep';
+use testapi;
+use partition_setup qw(create_new_partition_table addpart addlv);
+use version_utils 'is_storage_ng';
+
+sub run {
+    create_new_partition_table;
+    # create boot and 2 lvm partitions
+    addpart(role => 'raw', fsid => 'bios-boot', size => 2);
```

Comment:
Could you please update ticket for SLE to adjust this line for other architectures? We need prep boot on ppc64, boot/zipl for s390x and efi for UEFI (aarch64).